src/Symfony/Bundle/FrameworkBundle/Command/WorkerRunCommand.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace Symfony\Bundle\FrameworkBundle\Command;
+
+use Symfony\Component\Amqp\Worker\ConfigurableLoopInterface;
```

Comment:
should be `Symfony\Component\Worker\Loop\ConfigurableLooInterface`, going to fix it while reviewing commands, will try to add functional tests for them btw